examples/misc/logos2.py

```diff
@@ -20,10 +20,10 @@
 def get_font_properties():
     # The original font is Calibri, if that is not installed, we fall back
     # to Carlito, which is metrically equivalent.
-    if 'Calibri' in matplotlib.font_manager.findfont('Calibri:bold'):
+    if 'Calibri' in matplotlib.font_manager.findfont('Calibri:bold').keys():
```

Comment:
I don't think you need `.keys()`?

Reply:
ah that's right, I was probably searching for findfont usages within mpl and happen to change this eagerly.